install/kubernetes/experimental-install.yaml

```diff
@@ -1079,6 +1079,7 @@ spec:
     type: RollingUpdate
   template:
     metadata:
+      annotations:
```

Comment:
Just curious, why are these `annotations` keys explicitly declared here?

Reply:
It’s from changing the structure of the chart logic slightly and then running the `make` commands. Previously some of the annotation flow logic was a bit strange and in some cases only facilitated annotations being added explicitly by the user in values.yaml...it’s a byproduct of ‘fixing’ that.